onchain/rollups/contracts/dapp/CartesiDApp.sol

```diff
@@ -78,6 +74,15 @@ contract CartesiDApp is
     using Bitmask for mapping(uint256 => uint256);
     using LibOutputValidation for OutputValidityProof;
 
+    // @notice Raised when executing an already executed voucher.
```

Comment:
My previous comment applies to all custom errors, not just the one I'm highlighting with this very comment! :-)